gen/definitions/device_ipv6_static_route.yaml

```diff
@@ -0,0 +1,106 @@
+    write_only: true
+    description:
+      UUID of the interface which has been given by `interface_logical_name` (e.g.
+      fmc_device_physical_interface.example.id or fmc_device_subinterface.example.id).
+      The value is ignored, but the attribute itself is useful for ensuring that Terraform creates interface resource
+      before the static route resource (and destroys the interface resource only after the static route has been
+      destroyed).
+    mandatory: true
+    example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+    test_value: fmc_device_physical_interface.test.id
+  - model_name: selectedNetworks
+    tf_name: destination_networks
+    description: >-
+      Set of the destination networks matching this route (fmc_network, fmc_host, but not fmc_range).
+    mandatory: true
+    type: Set
+    attributes:
+      - model_name: id
+        type: String
+        description: UUID of the object (such as fmc_network.this.id, etc.).
+        id: true
+        example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+        test_value: fmc_network.this.id
+  - model_name: metricValue
+    type: Int64
+    description: >-
+      The cost of the route. The metric is used to compare routes among different routing protocols. The default
+      administrative distance for static routes is 1, giving it precedence over routes discovered by dynamic routing
+      protocols but not directly connected routes.
+    min_int: 1
+    max_int: 254
+    example: 254
+  - model_name: id
+    data_path: [gateway, object]
+    tf_name: gateway_object_id
+    description: >-
+      UUID of the next hop for this route (such as fmc_host.example.id). Exactly one of `gateway_object_id`
+      or `gateway_literal` must be present.
+    type: String
+    exclude_example: true
+    exclude_test: true
+  - model_name: value
+    data_path: [gateway, literal]
+    tf_name: gateway_literal
+    type: String
+    description: >-
+      The next hop for this route as an literal IPv6 address. Exactly one of `gateway_object_id`
+      or `gateway_literal` must be present.
+    example: 2024::1
+    minimum_test_value: '"2024::2"'
+  - model_name: isTunneled
+    type: Bool
+    description:
+      Indicates whether this route is a separate default route for VPN traffic. Should be used for default
+      route only, such as when the destination_networks points to a builtin object 'any-ipv6' or 'any-ipv4'. Useful
+      if you want VPN traffic to use a different default route than non-VPN traffic.
+      When a tunnel terminates on the device all traffic from it that cannot be routed using learned or static routes
+      is sent to this route. You can configure only one default tunneled gateway per device. ECMP for tunneled traffic
+      is not supported.
+    exclude_example: true
+    default_value: "false"
+    test_value: '"false"'
+
+test_prerequisites: |
+  variable "device_id" { default = null } // tests will set $TF_VAR_device_id
+
+  resource fmc_network this {
```

Comment:
Just cosmetic.. can we add the quotes for the resource type/name and name all the test resources "test".

Reply:
done, thanks

Reply:
I also made the tests to include `is_tunneled = true`